samples/bot-all-cards/csharp/BotAllCards/Dialogs/AllCards.cs

```diff
@@ -0,0 +1,157 @@
+            var paths = new[] { ".", "Resources", "heroCard.json" };
+            var thumbnailCardJson = File.ReadAllText(Path.Combine(paths));
+
+            var thumbnailCardAttachment = new Attachment()
+            {
+                ContentType = "application/vnd.microsoft.card.thumbnail",
+                Content = JsonConvert.DeserializeObject(thumbnailCardJson),
+            };
+
+            return thumbnailCardAttachment;
+        }
+
+        /// <summary>
+        /// Signin Card
+        /// </summary>
+        /// <returns></returns>
+        public static SigninCard GetSigninCard()
+        {
+            var signinCard = new SigninCard
+            {
+                Text = "BotFramework Sign-in Card",
+                Buttons = new List<CardAction>
+                {
+                    new CardAction(
+                        ActionTypes.OpenUrl, "Sign-in", value: "https://login.microsoftonline.com/") },
+            };
+
+            return signinCard;
+        }
+
+        /// <summary>
+        /// Card collections
+        /// </summary>
+        /// <returns></returns>
+        public static Attachment CollectionsCardAttachment()
+        {
+            var paths = new[] { ".", "Resources", "collectionsCard.json" };
+            var collectionsCardJson = File.ReadAllText(Path.Combine(paths));
+
+            var collectionsCardAttachment = new Attachment()
+            {
+                ContentType = "application/vnd.microsoft.card.adaptive",
+                Content = JsonConvert.DeserializeObject(collectionsCardJson),
+            };
+
+            return collectionsCardAttachment;
+        }
+
+        /// <summary>
+        /// Office 365 connector card
+        /// </summary>
+        /// <returns></returns>
+        public static Attachment Office365ConnectorCard()
+        {
+            var paths = new[] { ".", "Resources", "o365ConnectorCard.json" };
+            var OfficeCardJson = File.ReadAllText(Path.Combine(paths));
+
+            var OfficeCardAttachment = new Attachment()
+            {
+                ContentType = "application/vnd.microsoft.teams.card.o365connector",
+                Content = JsonConvert.DeserializeObject(OfficeCardJson),
+            };
+
+            return OfficeCardAttachment;
+        }
+
+        /// <summary>
+        /// OAuth Card
+        /// </summary>
+        /// <returns></returns>
+        public static OAuthCard GetOAuthCard()
+        {
+            var oauthCard = new OAuthCard
+            {
+                Text = "BotFramework OAuth Card",
+                ConnectionName = "YOUR-CONNECTION-NAME", // Replace with the name of your Azure AD connection.
+                Buttons = new List<CardAction> { new CardAction(ActionTypes.OpenUrl, "Sign In", value: "https://login.microsoftonline.com/") },
+            };
+
+            return oauthCard;
+        }
+
+        /// <summary>
+        /// List card
+        /// </summary>
+        /// <returns></returns>
```

Comment:
Please add proper summary for this method.

`/// <returns></returns>`
Why this is empty whereas you are returning something?

Reply:
Fixed